Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+                                         args=args,
+                                         parser_command=client.parser.parse_custom_whitelist)
+    validate_custom_whitelist(version=client.version, args=args, member_type='URL')
+    response = client.custom_whitelist_url_update_request(id=id,
+                                                          request_type=args.get('request_type'),
+                                                          request_url=args.get('request_url'),
+                                                          status=args.get('status'))
+    command_results = generate_simple_command_results(object_type=OutputTitle.CUSTOM_WHITELIST_URL.value,
+                                                      object_name=id,
+                                                      action=OutputTitle.UPDATED.value,
+                                                      response=response)
+
+    return command_results
+
+
+def custom_whitelist_parameter_create_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Create a custom whitelist parameter member.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+
+    validate_custom_whitelist(version=client.version, args=args, member_type='Parameter')
+    name = args['name']
+    response = client.custom_whitelist_parameter_create_request(name=name,
+                                                                name_type=args.get('name_type'),
+                                                                request_url_status=args.get('request_url_status'),
+                                                                request_type=args.get('request_type'),
+                                                                request_url=args.get('request_url'),
+                                                                domain_status=args.get('domain_status'),
+                                                                domain_type=args.get('domain_type'),
+                                                                domain=args.get('domain'))
+    member_id = get_object_id(client, response, 'itemName', name, client.custom_whitelist_list_request)
+
+    command_results = generate_simple_context_data_command_results('id', member_id, response,
+                                                                   OutputTitle.CUSTOM_WHITELIST_PARAMETER_CREATE.value,
+                                                                   'FortiwebVM.CustomGlobalWhitelist')
+
+    return command_results
+
+
+def get_object_data_before_update(client: Client,
+                                  value: str,
+                                  get_request: Callable,
+                                  args: Dict[str, Any],
+                                  parser_command: Callable,
```

Comment:
we use `_command` for XSOAR commands. rename this to `<...>_parser` based on what is parsed.